make-common.mk

```diff
@@ -92,6 +92,8 @@ tmp/image-%/.sentinel: docker-image-src/$(series)/Dockerfile docker-image-src/$(
 > mkdir -p $(@D)/local-package
 > cp docker-image-src/common/* $(@D)/local-package
 > cp $(filter %.sh,$^) $(@D)/local-package
+> NEO4JPLUGINS_OVERRIDE=docker-image-src/$(series)/neo4jlabs-plugins.json
+> if [ -e $${NEO4JPLUGINS_OVERRIDE} ]; then cp $${NEO4JPLUGINS_OVERRIDE} $(@D)/local-package; fi
```

Comment:
I'm overriding the `neo4j-labs-plugins.json` from the common folder for the `series` if that exists.